samples/GraphQL.DataLoader.Example/DealershipDbContext.cs

```diff
@@ -0,0 +1,83 @@
+using DataLoaderGql.Types;
+using Microsoft.Data.Sqlite;
+using Microsoft.EntityFrameworkCore;
+
+namespace DataLoaderGql;
+
+public class DealershipDbContext : DbContext
+{
+    public DbSet<Car> Cars => Set<Car>();
+    public DbSet<Salesperson> Salespeople => Set<Salesperson>();
+
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+    {
+        optionsBuilder
+            .UseSqlite(
+                new SqliteConnectionStringBuilder()
+                {
+                    Mode = SqliteOpenMode.ReadWriteCreate,
```

Comment:
Wouldn't it be better to use a `Memory` mode for easier sample running?

Reply:
Yeah, I completely forgot this "little" detail...

Reply:
also i personally prefer sqlite for a sample project.
As the Db gets automatically created, i don't see any harm to ease of use. And the User of the sample can even browse the little db, to maybe confirm how he thinks it all works.